internal/store/global_deployment.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store
+
+import (
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+//go:generate mockgen -destination=../mocks/mock_global_deployment.go -package=mocks github.com/mongodb/mongodb-atlas-cli/internal/store GlobalDeploymentDescriber
+
+type GlobalDeploymentDescriber interface {
+	GlobalDeployment(string, string) (*atlas.GlobalCluster, error)
+}
+
+func (s *Store) GlobalDeployment(projectID, instanceName string) (*atlas.GlobalCluster, error) {
```

Comment:
Let's keep the same name as the go client
```suggestion
func (s *Store) GlobalCluster(projectID, instanceName string) (*atlas.GlobalCluster, error) {
```